lib/reviewit/lib/reviewit/action/push.rb

```diff
@@ -48,6 +51,16 @@ def append_mr_id_to_commit mr_id
       end
     end
 
+    def rename_local_branch mr_id
+      branch = options[:branch]
```

Comment:
True. I've made change as you suggested.
btw: Thanks in advance. This is my first PR.